### ISSUE TITLE
cannnot import papermill

### ISSUE THREAD

**Issue by garland3** [nteract/papermill#510]
I cannot import papermill after installing it. 
```
>>> import papermill as pm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'papermill'
```

# How to recreate
```
(fastai2) [agarlan@egs00211 designstudy]$ conda create --name papermilltest python=3.7
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /ascldap/users/agarlan/anaconda3/envs/papermilltest

  added / updated specs:
    - python=3.7


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ld_impl_linux-64-2.34      |       h53a641e_4         616 KB  conda-forge
    pip-20.1.1                 |             py_1         1.1 MB  conda-forge
    setuptools-47.1.1          |   py37hc8dfbb8_0         647 KB  conda-forge
    xz-5.2.5                   |       h516909a_0         430 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         2.7 MB

The following NEW packages will be INSTALLED:

  _libgcc_mutex      conda-forge/linux-64::_libgcc_mutex-0.1-conda_forge
  _openmp_mutex      conda-forge/linux-64::_openmp_mutex-4.5-0_gnu
  ca-certificates    conda-forge/linux-64::ca-certificates-2020.4.5.1-hecc5488_0
  certifi            conda-forge/linux-64::certifi-2020.4.5.1-py37hc8dfbb8_0
  ld_impl_linux-64   conda-forge/linux-64::ld_impl_linux-64-2.34-h53a641e_4
  libffi             conda-forge/linux-64::libffi-3.2.1-he1b5a44_1007
  libgcc-ng          conda-forge/linux-64::libgcc-ng-9.2.0-h24d8f2e_2
  libgomp            conda-forge/linux-64::libgomp-9.2.0-h24d8f2e_2
  libstdcxx-ng       conda-forge/linux-64::libstdcxx-ng-9.2.0-hdf63c60_2
  ncurses            conda-forge/linux-64::ncurses-6.1-hf484d3e_1002
  openssl            conda-forge/linux-64::openssl-1.1.1g-h516909a_0
  pip                conda-forge/noarch::pip-20.1.1-py_1
  python             conda-forge/linux-64::python-3.7.6-h8356626_5_cpython
  python_abi         conda-forge/linux-64::python_abi-3.7-1_cp37m
  readline           conda-forge/linux-64::readline-8.0-hf8c457e_0
  setuptools         conda-forge/linux-64::setuptools-47.1.1-py37hc8dfbb8_0
  sqlite             conda-forge/linux-64::sqlite-3.30.1-hcee41ef_0
  tk                 conda-forge/linux-64::tk-8.6.10-hed695b0_0
  wheel              conda-forge/noarch::wheel-0.34.2-py_1
  xz                 conda-forge/linux-64::xz-5.2.5-h516909a_0
  zlib               conda-forge/linux-64::zlib-1.2.11-h516909a_1006


Proceed ([y]/n)? y


Downloading and Extracting Packages
ld_impl_linux-64-2.3 | 616 KB    | ##################################################################################################################################### | 100% 
pip-20.1.1           | 1.1 MB    | ##################################################################################################################################### | 100% 
setuptools-47.1.1    | 647 KB    | ##################################################################################################################################### | 100% 
xz-5.2.5             | 430 KB    | ##################################################################################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate papermilltest
#
# To deactivate an active environment, use
#
#     $ conda deactivate

(fastai2) [agarlan@egs00211 designstudy]$ conda activate papermilltest
(papermilltest) [agarlan@egs00211 designstudy]$ conda install papermill
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /ascldap/users/agarlan/anaconda3/envs/papermilltest

  added / updated specs:
    - papermill


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    async_generator-1.10       |             py_0          18 KB  conda-forge
    brotlipy-0.7.0             |py37h8f50634_1000         346 KB  conda-forge
    click-7.1.2                |     pyh9f0ad1d_0          64 KB  conda-forge
    cryptography-2.9.2         |   py37hb09aad4_0         622 KB  conda-forge
    pyyaml-5.3.1               |   py37h8f50634_0         185 KB  conda-forge
    pyzmq-19.0.1               |   py37hac76be4_0         519 KB  conda-forge
    six-1.15.0                 |     pyh9f0ad1d_0          14 KB  conda-forge
    urllib3-1.25.9             |             py_0          92 KB  conda-forge
    yaml-0.2.4                 |       h516909a_0          82 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         1.9 MB

The following NEW packages will be INSTALLED:

  ansiwrap           conda-forge/noarch::ansiwrap-0.8.4-py_0
  appdirs            conda-forge/noarch::appdirs-1.4.3-py_1
  async_generator    conda-forge/noarch::async_generator-1.10-py_0
  attrs              conda-forge/noarch::attrs-19.3.0-py_0
  black              conda-forge/linux-64::black-19.10b0-py37_0
  brotlipy           conda-forge/linux-64::brotlipy-0.7.0-py37h8f50634_1000
  cffi               conda-forge/linux-64::cffi-1.14.0-py37hd463f26_0
  chardet            conda-forge/linux-64::chardet-3.0.4-py37hc8dfbb8_1006
  click              conda-forge/noarch::click-7.1.2-pyh9f0ad1d_0
  cryptography       conda-forge/linux-64::cryptography-2.9.2-py37hb09aad4_0
  decorator          conda-forge/noarch::decorator-4.4.2-py_0
  entrypoints        conda-forge/linux-64::entrypoints-0.3-py37hc8dfbb8_1001
  idna               conda-forge/noarch::idna-2.9-py_1
  importlib-metadata conda-forge/linux-64::importlib-metadata-1.6.0-py37hc8dfbb8_0
  importlib_metadata conda-forge/noarch::importlib_metadata-1.6.0-0
  ipython_genutils   conda-forge/noarch::ipython_genutils-0.2.0-py_1
  jsonschema         conda-forge/linux-64::jsonschema-3.2.0-py37hc8dfbb8_1
  jupyter_client     conda-forge/noarch::jupyter_client-6.1.3-py_0
  jupyter_core       conda-forge/linux-64::jupyter_core-4.6.3-py37hc8dfbb8_1
  libsodium          conda-forge/linux-64::libsodium-1.0.17-h516909a_0
  mypy_extensions    conda-forge/linux-64::mypy_extensions-0.4.3-py37hc8dfbb8_1
  nbclient           conda-forge/noarch::nbclient-0.3.1-py_0
  nbformat           conda-forge/noarch::nbformat-5.0.6-py_0
  nest-asyncio       conda-forge/noarch::nest-asyncio-1.3.3-py_0
  papermill          conda-forge/linux-64::papermill-2.1.0-py37hc8dfbb8_0
  pathspec           conda-forge/noarch::pathspec-0.8.0-pyh9f0ad1d_0
  pycparser          conda-forge/noarch::pycparser-2.20-py_0
  pyopenssl          conda-forge/noarch::pyopenssl-19.1.0-py_1
  pyrsistent         conda-forge/linux-64::pyrsistent-0.16.0-py37h8f50634_0
  pysocks            conda-forge/linux-64::pysocks-1.7.1-py37hc8dfbb8_1
  python-dateutil    conda-forge/noarch::python-dateutil-2.8.1-py_0
  pyyaml             conda-forge/linux-64::pyyaml-5.3.1-py37h8f50634_0
  pyzmq              conda-forge/linux-64::pyzmq-19.0.1-py37hac76be4_0
  regex              fastai/linux-64::regex-2019.08.19-py37h516909a_0
  requests           conda-forge/noarch::requests-2.23.0-pyh8c360ce_2
  six                conda-forge/noarch::six-1.15.0-pyh9f0ad1d_0
  tenacity           conda-forge/linux-64::tenacity-6.2.0-py37hc8dfbb8_0
  textwrap3          conda-forge/noarch::textwrap3-0.9.2-py_0
  toml               conda-forge/noarch::toml-0.10.1-pyh9f0ad1d_0
  tornado            conda-forge/linux-64::tornado-6.0.4-py37h8f50634_1
  tqdm               conda-forge/noarch::tqdm-4.46.0-pyh9f0ad1d_0
  traitlets          conda-forge/linux-64::traitlets-4.3.3-py37hc8dfbb8_1
  typed-ast          conda-forge/linux-64::typed-ast-1.4.1-py37h516909a_0
  typing_extensions  conda-forge/noarch::typing_extensions-3.7.4.2-py_0
  urllib3            conda-forge/noarch::urllib3-1.25.9-py_0
  yaml               conda-forge/linux-64::yaml-0.2.4-h516909a_0
  zeromq             conda-forge/linux-64::zeromq-4.3.2-he1b5a44_2
  zipp               conda-forge/noarch::zipp-3.1.0-py_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
urllib3-1.25.9       | 92 KB     | ##################################################################################################################################### | 100% 
async_generator-1.10 | 18 KB     | ##################################################################################################################################### | 100% 
pyyaml-5.3.1         | 185 KB    | ##################################################################################################################################### | 100% 
six-1.15.0           | 14 KB     | ##################################################################################################################################### | 100% 
pyzmq-19.0.1         | 519 KB    | ##################################################################################################################################### | 100% 
cryptography-2.9.2   | 622 KB    | ##################################################################################################################################### | 100% 
click-7.1.2          | 64 KB     | ##################################################################################################################################### | 100% 
brotlipy-0.7.0       | 346 KB    | ##################################################################################################################################### | 100% 
yaml-0.2.4           | 82 KB     | ##################################################################################################################################### | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(papermilltest) [agarlan@egs00211 designstudy]$ python
Python 3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import papermill as pm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'papermill'
```    
I'm showing a  test conda environment, but I have the same problem on main environment. 

Perhaps this issues [https://github.com/puckel/docker-airflow/issues/421](url) is similar?

# Expect behavior
Well, I don't expect an import error.

**Comment by garland3** [comment]
I figured out the issue. Since, the command line `>>papermill` worked, but the python `import papermill` did not, I checked see which python was being run.   
`which python`
Which pointed to some system python and not my conda enviroment
`which papermill`
Showed papermill was in the correct place in my conda environment. 
`echo $PATH`    
Showed some system python would be found before my conda environment.     
So, I logged out of my terminal, and logged back in, and then checked path again. 
`echo $PATH`
And now my conda environment is first. So, I checked if `import papermill` works again, and it does!
```
base) [agarlan@egs00211 agarlan]$ conda activate papermilltest
(papermilltest) [agarlan@egs00211 agarlan]$ python
Python 3.7.6 | packaged by conda-forge | (default, Mar 23 2020, 23:03:20) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import papermill as pm
```

So, I guess this was some conda issue and not a papermill problem. I installed conda a few months ago. I guess it is possible I never refreshed the terminal since then.
